json/actions.json

```diff
@@ -83,98 +83,10 @@
     "data": {
-      "description": "Orari lezioni di oggi (tuo anno)",
-      "command": "lezionioggi",
-      "noYear": "Questo non sembra essere un gruppo generale per nessun anno di corso. Per usare questo comando da qui, devi specificare /lezionioggi1, /lezionioggi2 o /lezionioggi3."
-    }
-  },
-  "lezionioggi1": {
-    "type": "todayLectures",
-    "data": {
-      "description": "Orari lezioni di oggi (1\u00b0 anno)",
-      "course": {
-        "year": 1,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di oggi (1\u00b0 anno):</b>\n",
-      "fallbackText": "Non ci sono lezioni oggi. SMETTILA DI PRESSARMI"
-    }
-  },
-  "lezionioggi2": {
-    "type": "todayLectures",
-    "data": {
-      "description": "Orari lezioni di oggi (2\u00b0 anno)",
-      "course": {
-        "year": 2,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di oggi (2\u00b0 anno):</b>\n",
-      "fallbackText": "Non ci sono lezioni oggi. SMETTILA DI PRESSARMI"
-    }
-  },
-  "lezionioggi3": {
-    "type": "todayLectures",
-    "data": {
-      "description": "Orari lezioni di oggi (3\u00b0 anno)",
-      "course": {
-        "year": 3,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di oggi:</b>\n",
-      "fallbackText": "Non ci sono lezioni oggi. SMETTILA DI PRESSARMI"
-    }
-  },
-  "lezionidomani": {
-    "type": "yearly",
-    "data": {
-      "description": "Orari lezioni di domani (tuo anno)",
-      "command": "lezionidomani",
-      "noYear": "Questo non sembra essere un gruppo generale per nessun anno di corso. Per usare questo comando da qui, devi specificare /lezionidomani1, /lezionidomani2 o /lezionidomani3."
-    }
-  },
-  "lezionidomani1": {
-    "type": "tomorrowLectures",
-    "data": {
-      "description": "Orari lezioni di domani (1\u00b0 anno)",
-      "course": {
-        "year": 1,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di domani (1\u00b0 anno):</b>\n",
-      "fallbackText": "Non ci sono lezioni domani. SMETTILA DI PRESSARMI"
-    }
-  },
-  "lezionidomani2": {
-    "type": "tomorrowLectures",
-    "data": {
-      "description": "Orari lezioni di domani (2\u00b0 anno)",
-      "course": {
-        "year": 2,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di domani (2\u00b0 anno):</b>\n",
-      "fallbackText": "Non ci sono lezioni domani. SMETTILA DI PRESSARMI"
-    }
-  },
-  "lezionidomani3": {
-    "type": "tomorrowLectures",
-    "data": {
-      "description": "Orari lezioni di domani (3\u00b0 anno)",
-      "course": {
-        "year": 3,
-        "type": "laurea",
-        "name": "informatica"
-      },
-      "title": "<b>Lezioni di domani (3\u00b0 anno):</b>\n",
-      "fallbackText": "Non ci sono lezioni domani. SMETTILA DI PRESSARMI"
+      "description": "Orari lezioni"

```

Comment:
Anche se non abbiamo al momento in programma di localizzare il bot in inglese (per chi studia AI in tale lingua), penso sia comunque una buona abitudine mantenere separata logica e stringhe da inviare come messaggio. Per questo motivo, non rimuoverei i campi "title" e "fallbackText" (puoi inserire i segnaposto dentro a "title" siccome hai generalizzato a vari giorni/curricula/anni).